Dockerfile

```diff
@@ -1,11 +1,20 @@
 FROM dmscid/lewis-depends:latest
 
+# Copying this separately from the rest of lewis allows the
+# pip install step to be cached until the requirements change.
+# Rebuilding is faster and doesn't require an internet connection.
+COPY requirements.txt /lewis/
+
+# Install any missing PIP requirements, clear PIP cache, and delete all compiled Python objects
+RUN pip install -r /lewis/requirements.txt && \
+    rm -rf /root/.cache/pip/* && \
+    find \( -name '*.pyc' -o -name '*.pyo' \) -exec rm -rf '{}' +
+
 # Lewis source is pulled in from current directory (note .dockerignore filters out some files)
 COPY . /lewis
 
-# Install any missing PIP requirements, clear PIP cache, and delete all compiled Python objects
+# Install lewis in-place and remove compiled Python objects
 RUN pip install -e /lewis && \
```

Comment:
Ah, that's right, I forgot they're completely gone now.

In that case it might be better to use `pip install -e` just for the `lewis` and `lewis-control` "shortcuts".

Alternatively, you can execute folders containing a `__main__.py` like this:
```
~/Dev/lewis$ python src/lewis
```
But on that note I think our __main__.py has a mistake in it... shouldn't this:
```python
from lewis.scripts.run import run_simulation
```
... be this instead...
```python
from scripts.run import run_simulation
```
... since we're referencing `src/lewis/scripts/run.py` from `src/lewis/__main__.py`?